api_tests/config.py

```diff
@@ -22,10 +22,11 @@ def parse():
 
     # Make sure config file has all required keys
     missing_key = verify_keys(config_dict, [
-      'url',
-      'access_token',
-      'snippet',
-      'user_id',
+        'url',
+        'access_token',
+        'snippet',
+        'user_id',
+        'invalid_snippet_key',
```

Comment:
It seems to me that this should be a hard coded value local to the specific function. It is not something that is used widely, across all of the tests.